ztp/policygenerator-kustomize-plugin/testPolicyGenTemplate/site-du-sno-1-ranGen.yaml

```diff
@@ -99,3 +99,8 @@ spec:
         storage:
           pvc:
             claim: "image-registry-pvc"
+    - fileName: HardwareEventRedfishSecret.yaml
+      policyName: "redfish-secret-cnfde10"
+      stringData:
+        # cnfde10
+        hostaddr: 10.16.231.97
```

Comment:
Is this overridable while deploying?

Reply:
I expect these to be overwritten with lab specific values by the person deploying the cluster.

Reply:
Deleted this. I will update documentation to let customer manually create secret like this:
```
oc -n openshift-hw-events create secret generic redfish-basic-auth --from-literal=username=$BMC_USERNAME --from-literal=password=$BMC_PASSWORD --from-literal=hostaddr="BMC_HOST_IP_ADDR"
```